static/app/views/eventsV2/table/columnEditCollection.tsx

```diff
@@ -549,7 +549,7 @@ class ColumnEditCollection extends React.Component<Props, State> {
 
 const RowContainer = styled('div')`
   display: grid;
-  grid-template-columns: ${space(3)} 1fr ${space(3)};
+  grid-template-columns: ${space(3)} 1fr 40px;
```

Comment:
the size was not correct here and button was overflowing 

![image](https://user-images.githubusercontent.com/29228205/154966940-2c3eece8-0011-4908-bc34-2316679ab745.png)